schemas/questions.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "required": [
+      "question",
+      "description",
+      "type"
+    ],
+    "properties": {
+      "question": {
+        "type": "string"
+      },
+      "description": {
+        "type": "string"
+      },
+      "validation": {
+        "type": "string"
+      },
+      "type": {
+        "type": "string",
+        "enum": [
+          "FREESINGLE",
+          "FREEMULTIPLE",
+          "CHOICESINGLE",
+          "CHOICEMULTIPLE"
+        ]
+      },
+      "options": {
```

Comment:
could you make options a required field when type:choice* is selected? 

you could use the if/then/else method in a newer version of the json schema. I do not think dependentRequired helps with this since it can not change based on values only on fields

Reply:
I believe depedentRequired applies to 'if a field is present', not 'if a field has value X':
https://json-schema.org/understanding-json-schema/reference/conditionals

It does seem possible using if-then or anyOf constructions. I am in favor of adding that later, as it makes the schema more complex (bad reason) and we're not sure if this is the final schema we want (better reason).